### PR TITLE
R116

### PR DESCRIPTION
**Pull request by chuwy**
No patches, new schemas only.

### CODE REVIEW

File: schemas/org.ietf/http_client_hints/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
```
Comment (jbeemster) [inline schemas/org.ietf/http_client_hints/jsonschema/1-0-0:3]: Do we want a description on this schema to tell us what it is for?  There are also no descriptions on any of the fields..

**Comment by chuwy** [comment]
Agree with Josh. Hey @paulboocock, could you please add descriptions to the schema? Feel free to either create a new branch or force-push

**Comment by paulboocock** [comment]
> Agree with Josh. Hey @paulboocock, could you please add descriptions to the schema? Feel free to either create a new branch or force-push 

Yeah, that was a little lazy of me.
Added descriptions now :-)

**Comment by chuwy** [comment]
Thanks Paul! @jbeemster should be ready now!

**Pull request merged**